dusk/grammar.py

```diff
@@ -250,14 +250,52 @@ def statements(self, py_stmts: t.List, in_stencil_root_scope: bool = False):
         return sir_stmts
 
     @transform(
-        Assign(
-            targets=FixedList(Capture(expr).to("lhs")),
-            value=Capture(expr).to("rhs"),
-            type_comment=None,
+        OneOf(
+            Assign(
+                targets=FixedList(Capture(expr).to("lhs")),
+                value=Capture(expr).to("rhs"),
+                type_comment=None,
+            ),
+            AugAssign(
+                target=Capture(expr).to("lhs"),
+                op=Capture(operator).to("op"),
+                value=Capture(expr).to("rhs"),
+            ),
         ),
     )
-    def assign(self, lhs: expr, rhs: expr):
-        return make_assignment_stmt(self.expression(lhs), self.expression(rhs))
+    def assign(self, lhs: expr, rhs: expr, op: t.Optional[operator] = None):
+
+        py_assign_op_to_sir_assign_op = {
+            Add: "+=",
+            Sub: "-=",
+            Mult: "*=",
+            Div: "/=",
+            Mod: "%=",
+            LShift: "<<=",
+            RShift: ">>=",
+            BitOr: "|=",
+            BitXor: "^=",
+            BitAnd: "&=",
+        }
+
+        if op is None:
+            op = "="
+
+        elif isinstance(op, Pow):
+            op = "="
+            rhs = make_fun_call_expr(
+                "gridtools::dawn::math::pow",
```

Comment:
```suggestion
                "math::pow",
```

Reply:
I took this from here: https://github.com/dawn-ico/dusk/blob/c8220ce22895bf4094a8d18ad025254b4708c397/dusk/grammar.py#L697

Actually, matthias changed this in his PR: https://github.com/dawn-ico/dusk/pull/26/commits/320b6a4b75ab276ddd355546f74ddc67466b9728 #26 

Not sure what's the right approach here. We should probably add an issue to change all of these once SIR has proper support for them.